css/smpte.css

```diff
@@ -288,6 +288,42 @@ tbody {
   text-align: left;
 }
 
+td.center-cell {
+  text-align: center;
+}
+
+table.col-1-center td:nth-child(1) {
```

Comment:
Shouldn't this be `tr td:nth-child(n)` as in nth td child of a tr?

Reply:
Apparently not, from what I'm reading it's always the 2nd `td` in each `tr` regardless. It appears to have the same effect either way. But I"ll change this as it make more sense to me as well that way.

Reply:
Addressed in https://github.com/SMPTE/html-pub/pull/135/commits/3ebfe8008d1fcda9c4376ea15b61704faf265698